components/tfx/Evaluator/component.py

```diff
@@ -0,0 +1,128 @@
+from kfp.components import InputPath, OutputPath
+
+
+def Evaluator(
+    examples_path: InputPath('Examples'),
+    model_exports_path: InputPath('Model'),
+    #model_path: InputPath('Model'),
```

Comment:
Can we use model_path from the very beginning. I think TFX is migrating to that.

Reply:
It's a bit non-trivial since the `ComponentSpec.INPUTS` uses the old names.
I'll solve this issue in another PR.